### ISSUE TITLE
[HOLD #12603][Image][Performance] Avatar images in a report are re-fetched when we hover on the message

### ISSUE THREAD

**Issue by youssef-lr** [Expensify/App#11393]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## What performance issue do we need to solve?
We need to figure out why the component re-renders even though nothing changes in the state. This could possibly be due to a parent component re-rendering which makes the child component `Avatar` re-render as well. 

https://user-images.githubusercontent.com/9680864/192889584-6ccdd960-f04c-4a25-8654-940afb1cd232.mov

## What is the impact of this on end-users?
TBD.

## List any benchmarks that show the severity of the issue
TBD.

## Proposed solution (if any)
TBD.

## List any benchmarks after implementing the changes to show impacts of the proposed solution (if any)
TBD.

## Platform:
Where is this issue occurring?

- [x] Web
- [x] iOS
- [x] Android
- [x] Desktop App
- [x] Mobile Web

**Comment by trjExpensify** [comment]
👋 @youssef-lr - is this one held on a plan for `Image-related feature request and improvements` or does it need a volunteer to work on it? CC: @roryabraham

**Comment by Beamanator** [comment]
Good question, I'd be willing to help look into this if it's not on hold

**Comment by roryabraham** [comment]
I think this should be on hold for https://github.com/Expensify/App/issues/10894, but that one's up-for-grabs @Beamanator 😉

**Comment by trjExpensify** [comment]
Go on, @Beamanator. You know you want to.. 😏 

Alrighty, I've popped a hold in the title here for now.

**Comment by trjExpensify** [comment]
You da man! 🏊‍♂️

**Comment by Beamanator** [comment]
Still on hold, making progress on image improvements, see tracking issue

**Comment by Beamanator** [comment]
Still on hold, will get to this soon hopefully

**Comment by trjExpensify** [comment]
Coming from [this thread](https://expensify.slack.com/archives/C035J5C9FAP/p1669032019939069?thread_ts=1669028674.179999&cid=C035J5C9FAP), I'm going to assign this to @thomas-coldwell as a follow-up to the caching PR.

**Comment by thomas-coldwell** [comment]
Hi, yes I'll square this off once the image caching PR is merged 🙌

**Comment by trjExpensify** [comment]
👋 Are we still held here as the image caching PR was reverted?

**Comment by trjExpensify** [comment]
We're still on hold here for the caching PR.

**Comment by trjExpensify** [comment]
Still on hold for the caching PRs.

**Comment by trjExpensify** [comment]
Still held on the web caching PR.

**Comment by trjExpensify** [comment]
No change. Web caching PR is close though!

**Comment by trjExpensify** [comment]
Still held. Fork PR is merged, and the App PR is in review. Latest on the held issue is here: https://github.com/Expensify/App/issues/12603#issuecomment-1410057871

**Comment by trjExpensify** [comment]
Still held on the linked App PR above.

**Comment by trjExpensify** [comment]
No change. Blocked by resolving CORS issues to unblock web caching.

**Comment by Beamanator** [comment]
Still on hold

**Comment by trjExpensify** [comment]
Same, caching is still blocking this one.

**Comment by Beamanator** [comment]
Still held

**Comment by trjExpensify** [comment]
Commented on #12603 to update that issue's hold to the correct place, as the issue it's pointing to as being held on has merged.

**Comment by trjExpensify** [comment]
Same, Melv.

**Comment by Beamanator** [comment]
Melv, same.

**Comment by trjExpensify** [comment]
No change yet!

**Comment by Beamanator** [comment]
same

**Comment by trjExpensify** [comment]
No change, server migration comes first then image cashing Melv.

**Comment by Beamanator** [comment]
still held

**Comment by trjExpensify** [comment]
Still working on web image caching!

**Comment by Beamanator** [comment]
same ^

**Comment by trjExpensify** [comment]
In light of https://github.com/Expensify/App/issues/32703 being fixed, the https://github.com/Expensify/App/pull/34505 is in review.

**Comment by trjExpensify** [comment]
New PRs are in the works and a bit of a wider plan. :)

**Comment by trjExpensify** [comment]
@youssef-lr @thomas-coldwell @Beamanator with all the performance improvements as of late, I think we took care of this one. I just tried to reproduce on a chat I have with a bunch of attachments/preview components and hovering over them doesn't re-fetch anything in the `Network` tab. 

Going to close this out, but if anyone disagrees, feel free to reopen!
